controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/billing/BillingApiHandlerV2Test.java

```diff
@@ -258,6 +258,14 @@ void lists_accepted_countries() {
         tester.assertJsonResponse(req, new File("accepted-countries.json"));
     }
 
+    @Test
+    void summarize_bill() {
+        var req = request("/billing/v2/accountant/bill/id-1/summary?keys=plan,architecture")
+                .roles(Role.hostedAccountant());
+        tester.assertResponse(req, """
+                {"id":"BillId{value='id-1'}","summary":[{"key":{"plan":"paid","architecture":"Optional.empty"},"summary":{"cpu":{"cost":"0","hours":"0"},"memory":{"cost":"0","hours":"0"},"disk":{"cost":"0","hours":"0"},"gpu":{"cost":"0","hours":"0"}}}]}""");
```

Comment:
`null` instead of `Optional.empty`? Missing unwrapping of `id`.

Reply:
Good point. Will fix.